### ISSUE TITLE
git appraise

### ISSUE THREAD

**Issue by Profpatsch** [neithernut/git-dit#7]
https://github.com/google/git-appraise

Maybe relevant, seems to have a different goal.

**Comment by matthiasbeyer** [comment]
Yes, we saw that already and we think that it is nice, but it focuses on code review, not on issue tracking. And it stores data in JSON, afaik.
We have to think hard about whether we want such things.

**Comment by Profpatsch** [comment]
> We have to think hard about whether we want such things (and we do not want JSON, that's clear).

I think the “empty commit” idea is the way to go.
Though the format of messages should be fixed a bit more, was thinking of using MIME, since that’s a very well-understood, widely implemented and extensible standard for messages.
It has lots of baggage, but I think it’s possible to define a subset without all the legacy (e.g. specify UTF-8 as only valid encoding).

**Comment by matthiasbeyer** [comment]
I think that's basically what we do with the tags.

**Comment by Profpatsch** [comment]
Is it possible to add multiple files with mimetype? Is it possible to attach other messages? Are headers standardized (incl encoding)?

MIME does a lot more than a few headers. For mapping it to existing mail infrastructure, it will be vital to do all these things, so why not just go with the existing standard?

**Comment by neithernut** [comment]
Good point.

However, I'd rather have files stored in the tree. It greatly simplifies accessing those files via regular git tools.
This is, of course, only possible as long as it doesn't conflict with other tree contents. However, I expect files to be only attached to bug-reports, maybe feature requests, but never to patch-sets.
